### PR TITLE
exp/tools/dump-ledger-state, services/horizon/docker: Update quorum set in stellar core docker containers

### PR DESCRIPTION
**Pull request by tamirms**
<details>
  <summary>PR Checklist</summary>
  
### PR Structure

* [x] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [x] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [x] This PR's title starts with name of package that is most changed in the PR, ex.
  `services/friendbot`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
* [ ] I've updated any docs ([developer docs](https://www.stellar.org/developers/reference/), `.md`
  files, etc... affected by this change). Take a look in the `docs` folder for a given service,
  like [this one](https://github.com/stellar/go/tree/master/services/horizon/internal/docs).

### Release planning

* [ ] I've updated the relevant CHANGELOG ([here](services/horizon/CHANGELOG.md) for Horizon) if
  needed with deprecations, added features, breaking changes, and DB schema changes.
* [ ] I've decided if this PR requires a new major/minor version according to
  [semver](https://semver.org/), or if it's mainly a patch change. The PR is targeted at the next
  release branch if it's not a patch change.
</details>

### Summary

Update the quorum set for the stellar core configuration in the dump ledger state job and in the docker-compose container.

### CODE REVIEW

File: exp/tools/dump-ledger-state/stellar-core.cfg [hunk 1]
```diff
@@ -6,49 +6,133 @@ DATABASE="postgresql://dbname=core host=localhost user=circleci"
...
 ]
 
-# Stellar.org history store
-[HISTORY.sdf1]
-get="curl -sf http://history.stellar.org/prd/core-live/core_live_001/{0} -o {1}"
+[HISTORY.cache]
+get="cp /opt/stellar/history-cache/{0} {1}"
+
+[[HOME_DOMAINS]]
+HOME_DOMAIN="sdf"
```
Comment (bartekn) [inline exp/tools/dump-ledger-state/stellar-core.cfg]: Can you update it with latest changes in https://github.com/stellar/docker-stellar-core-horizon/commit/c0aa13c5245ffb52bf40d15c524aa48c04e1c9c7?

File: services/horizon/docker/stellar-core-pubnet.env [hunk 2]
```diff
@@ -5,9 +5,9 @@ compat_minimal_HTTP_PORT=11626
 
 CATCHUP_RECENT=8640
 
-KNOWN_PEERS=["core-live-a.stellar.org:11625", "core-live-b.stellar.org:11625", "core-live-c.stellar.org:11625", "confucius.strllar.org", "stellar1.bitventure.co", "stellar.256kw.com"]
+KNOWN_PEERS=["core-live-a.stellar.org:11625", "core-live-b.stellar.org:11625", "core-live-c.stellar.org:11625"]
```
Comment (bartekn) [inline services/horizon/docker/stellar-core-pubnet.env]: I think it was completely removed after recent changes in stellar/quickstart repo, it's using validators as known peers.

File: services/horizon/docker/stellar-core-pubnet.env [hunk 3]
```diff
@@ -5,9 +5,9 @@ compat_minimal_HTTP_PORT=11626
...
 
-VALIDATORS=["GCGB2S2KGYARPVIA37HYZXVRM2YZUEXA6S33ZU5BUDC6THSB62LZSTYH", "GAOO3LWBC4XF6VWRP5ESJ6IBHAISVJMSBTALHOQM2EZG7Q477UWA6L7U", "GCJCSMSPIWKKPR7WEPIQG63PDF7JGGEENRC33OKVBSPUDIRL6ZZ5M7OO", "GCM6QMP3DLRPTAZW2UZPCPX2LF3SXWXKPMP3GKFZBDSF3QZGV2G5QSTK", "GABMKJM6I25XI4K7U6XWMULOUQIQ27BCTMLS6BYYSOWKTBUXVRJSXHYQ"]
+VALIDATORS=["GCGB2S2KGYARPVIA37HYZXVRM2YZUEXA6S33ZU5BUDC6THSB62LZSTYH", "GAK6Z5UVGUVSEK6PEOCAYJISTT5EJBB34PN3NOLEQG2SUKXRVV2F6HZY", "GDKWELGJURRKXECG3HHFHXMRX64YWQPUHKCVRESOX3E5PM6DM4YXLZJM", "GCM6QMP3DLRPTAZW2UZPCPX2LF3SXWXKPMP3GKFZBDSF3QZGV2G5QSTK", "GABMKJM6I25XI4K7U6XWMULOUQIQ27BCTMLS6BYYSOWKTBUXVRJSXHYQ"]
```
Comment (bartekn) [inline services/horizon/docker/stellar-core-pubnet.env]: I think that after updating to the latest validators format quorum is generated automatically so no need for this param here.

**Comment by tamirms** [comment]
@bartekn I have updated exp/tools/dump-ledger-state/stellar-core.cfg  with the latest changes in https://github.com/stellar/docker-stellar-core-horizon/commit/c0aa13c5245ffb52bf40d15c524aa48c04e1c9c7

However, I could not use the format in the stellar core configuration in services/horizon/docker/. The stellar core docker image does not allow you to easily override the stellar core config file (see https://github.com/stellar/docker-stellar-core/issues/24).

The only way you can customize the stellar core docker image is by providing environment variables which are interpreted by confd ( https://github.com/stellar/docker-stellar-core/blob/master/confd/templates/stellar-core.cfg.tmpl ). The confd template in https://github.com/stellar/docker-stellar-core does not provide a way to define validators using the new format

**Pull request merged**